tests/components/fitbit/test_sensor.py

```diff
@@ -242,32 +242,52 @@ async def test_device_battery_level(
 
     state = hass.states.get("sensor.charge_2_battery")
     assert state
-    assert state.state == "Medium"
+    assert state.state == "60"
```

Comment:
Will this mean that existing users will have the battery entities with entity_ids such as `sensor.charge_2_battery_2` for the integer measurement and `sensor.charge_2_battery` for the existing string measurement?

Maybe we shouldn't swap the names?

Reply:
Reverting this part.